cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -152,24 +154,26 @@ private def artifactCleanup(int months, String[] repos, log, paceTimeMS, dryRun
                 cntNoDeletePermissions++
             }
             if (dryRun) {
-                    log.info "Found $it, $cntFoundArtifacts/$artifactsCleanedUp.size total $bytesFound bytes"
-                    log.info "\t==> currentUser: ${security.currentUser().getUsername()}"
-                    log.info "\t==> canDelete: ${security.canDelete(it)}"
+                    log.debug "Found $it, $cntFoundArtifacts/$artifactsCleanedUp.size total $bytesFound bytes"
+                    log.debug "\t==> currentUser: ${security.currentUser().getUsername()}"
+                    log.debug "\t==> canDelete: ${security.canDelete(it)}"
                     if (!checkName(keepArtifacts, keepArtifactsRegex, it)) {
-                        log.info "\t==> protected by regex: ${keepArtifactsRegex}"
+                        log.debug "\t==> protected by regex: ${keepArtifactsRegex}"
                     }
             } else {
                 if (security.canDelete(it) && (checkName(keepArtifacts, keepArtifactsRegex, it))) {
-                    log.info "Deleting $it, $cntFoundArtifacts/$artifactsCleanedUp.size total $bytesFound bytes"
+                    bytesRemoved += repositories.getItemInfo(it)?.getSize()
+                    log.debug "Deleting $it, [$cntFoundArtifacts/$artifactsCleanedUp.size]. Removed $bytesRemoved of total $bytesFound bytes"
                     repositories.delete it
                 } else {
-                    log.info "Can't delete $it (user ${security.currentUser().getUsername()} has no delete permissions), " +
+                    log.debug "Can't delete $it (user ${security.currentUser().getUsername()} has no delete permissions), " +
                             "$cntFoundArtifacts/$artifactsCleanedUp.size total $bytesFound bytes"
+                    bytesFoundWithNoDeletePermission += repositories.getItemInfo(it)?.getSize()
```

Comment:
This seems to be  already counted on: https://github.com/tt-gf/artifactory-user-plugins/blob/72fb322abe6a8b77348553a2d78aa2ab5b0310bb/cleanup/artifactCleanup/artifactCleanup.groovy#L153

Reply:
Yes but the line I am referring is actually already counting it when there are no permissions. Only difference is if a user has delete permissions, but the `(checkName(keepArtifacts, keepArtifactsRegex, it)` is also not satisfied. In that situation I would expect us tracking a different bucket instead of counting for no delete permissions.